tests/e2e/cluster/up.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+
+source "${script_dir}/../lib.sh"
+
+# Run kubeadm init and KUBECONFIG is exported on success.
+#
+init_kubeadm() {
+	local kubeadm_config_file="/etc/kubeadm/kubeadm.conf"
+	# Bootstrap the control-plane node.
+	kubeadm init --config "${kubeadm_config_file}"
```

Comment:
I've always used the `-pod-network-cidr` and `--cri-socket` args, which don't appear to be in your config file.

Reply:
Those options are passed via configuration file: https://github.com/confidential-containers/operator/pull/63/files#diff-f503b1c550a96bc32769671319e42cf289e6a2bcc203d5d16d0fd00b623b8f85

But now that you mentioned, I will double-check those configurations are really taken effect.